cdap-api-spark/src/main/java/co/cask/cdap/api/spark/JavaSparkExecutionContext.java

```diff
@@ -219,6 +221,47 @@
                                                       Class<K> keyType, Class<V> valueType);
 
   /**
+   * Creates a {@link JavaPairRDD} that represents all events from the given stream.
+   * The first entry in the pair is a {@link Long}, representing the
+   * event timestamp, while the second entry is a {@link GenericStreamEventData},
+   * which contains data decoded from the stream event body base on
+   * the given {@link FormatSpecification}.
+   *
+   * @param streamName name of the stream
+   * @param formatSpec the {@link FormatSpecification} describing the format in the stream
+   * @param <T> value type
+   * @return a new {@link JavaPairRDD} instance that reads from the given stream.
+   * @throws DatasetInstantiationException if the Stream doesn't exist
```

Comment:
`StreamNotFound` is an internal exception. Besides we treat Stream as a special type of Dataset at the API level.